backend/config.coffee

```diff
@@ -79,6 +79,20 @@ class Config
         else
             null
 
+    # Set the pull or push mode for this device
+    # It wan throw an exception if the mode is not compatible with the last
+    # mode used!
+    setMode: (mode, deviceName) ->
+        deviceName ?= @getDefaultDeviceName()
+        if deviceName and @devices[deviceName]
+            old = @devices[deviceName].mode
+            return true if old is mode
```

Comment:
Please do not use return in the middle of a function. It breaks indentation and makes the function "unpredictable".